bin/install-data

```diff
@@ -160,9 +174,9 @@ sub mapColValues {
 
   if ($valueFromFile eq '@USER_DATASET_ID@') { return "'$userDatasetId'"; }
   if ($valueFromFile eq '@STUDY_ID@') { return $platform eq 'Oracle'? "$schema.study_sq.nextval" : "nextval($schema.study_sq)"; }
-  if ($valueFromFile eq '@MODIFICATION_DATE@') { return "SYSDATE" ; }
+  if ($valueFromFile eq '@MODIFICATION_DATE@') { return "current_timestamp" ; }
   if ($valueFromFile eq '@ENTITY_TYPE_GRAPH_ID@') { return $platform eq 'Oracle'? "$schema.entitytypegraph_sq.nextval" : "nextval($schema.entitytypegraph_sq)"; }
-  if (($colType eq 'SQL_NUMBER' or $colType eq 'SQL_DATE') and $valueFromFile eq "")  { return "NULL"; }
+  if (($colType eq 'SQL_NUMBER' or $colType eq 'SQL_DATE' or $colType eq 'SQL_VARCHAR') and $valueFromFile eq "")  { return "NULL"; }
```

Comment:
Not sure if this is ok. It's needed to ensure columns that EDA expects to be null to be null instead of `""`, but I'm not sure if this has caused collateral damage for some other use-case